src/main/java/org/embulk/output/bigquery_java/BigqueryClient.java

```diff
@@ -308,6 +335,162 @@ public void onGiveup(Exception firstException, Exception lastException) throws R
         }
     }
 
+    public JobStatistics.QueryStatistics merge(String sourceTable, String targetTable, List<String> mergeKeys, List<String> mergeRule) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("MERGE ");
```

Comment:
https://cloud.google.com/bigquery/docs/reference/standard-sql/dml-syntax#merge_statement

Reply:
SQL will look like this.
```
MERGE `test_by_sano`.`output` T
USING `test_by_sano`.`LOAD_TEMP_b7c8436b_de2e_458b_8ab0_faf969b1ed9c_output` S
   ON T.`key_long` = S.`key_long`
  AND T.`key_string` = S.`key_string`
 WHEN MATCHED THEN
    UPDATE SET
      T.`key_long` = S.`key_long`,
      T.`key_string` = S.`key_string`,
      T.`test_boolean` = S.`test_boolean`,
      T.`test_long` = S.`test_long`,
      T.`test_double` = S.`test_double`,
      T.`test_string` = S.`test_string`,
      T.`test_timestamp` = S.`test_timestamp`,
      T.`test_json` = S.`test_json`
 WHEN NOT MATCHED THEN
    INSERT (
      `key_long`,
      `key_string`,
      `test_boolean`,
      `test_long`,
      `test_double`,
      `test_string`,
      `test_timestamp`,
      `test_json`)
    VALUES (
      `key_long`,
      `key_string`,
      `test_boolean`,
      `test_long`,
      `test_double`,
      `test_string`,
      `test_timestamp`,
      `test_json`)
```

Reply:
NOTE:
`MERGE` statement initiates an implicit transaction and are automatically committed.
https://cloud.google.com/bigquery/docs/data-manipulation-language#limitations
```
Each DML statement initiates an implicit transaction,
which means that changes made by the statement are automatically committed at the end of each successful DML statement.
```
https://cloud.google.com/bigquery/docs/transactions#statements_supported_in_transactions
https://cloud.google.com/bigquery/docs/multi-statement-queries#collect_multi_statement_query_info
https://cloud.google.com/bigquery/docs/information-schema-jobs#multi-statement_query_job